app.py

```diff
@@ -75,6 +75,27 @@ def skill():
         return jsonify({})
 
     if request.method == 'POST':
-        return jsonify({})
+        return add_skill()
 
     return jsonify({})
+
+
+def add_skill():
+    
+    '''
+     Add a skill using POST method
+     
+    '''
+    req = request.get_json()
+
+    required_fields = {"name":"string", "proficiency":"string", "logo":"string"}
+
+    missing_fields = [field for field in required_fields.keys() if field not in req]
+    
+    if missing_fields:
+        return jsonify({"error": "Missing required field(s): " + ", ".join(missing_fields)}), 400
```

Comment:
Great stuff, nice to see you are checking for errors! :)
The line results long and difficult to parse; maybe you can try to split it on multiple lines?